packages/astro/src/vite/error-page.ts

```diff
@@ -17,15 +17,24 @@ export const errorPageInjectionPlugin: (options: ErrorPagePluginOptions) => Plug
       const initSnippet = buildSpotlightErrorPageSnippet({
         ...options,
         // Astro's toolbar isn't available in the error page
-        showTriggerButton: true,
+        showTriggerButton: false,
+        fullPage: true,
         // don't wait for the window.load event to be fired because in the error page,
         // this already happened before spotlight is initialized
         injectImmediately: true,
+        showRecentError: true,
       });
 
-      const modifiedCode = `${code}\n${initSnippet}\n`;
+      // Checking if there is a ErrorOverlay class added by Astro
```

Comment:
This definitely feels quite fragile. Do we do this kind of find/replace on actual JS/TS code somewhere?